kong/db/strategies/off/init.lua

```diff
@@ -1,26 +1,31 @@
-local declarative_config = require "kong.db.schema.others.declarative_config"
-local workspaces = require "kong.workspaces"
+local declarative_config = require("kong.db.schema.others.declarative_config")
```

Comment:
Is there somewhere where I can read about the changes to declarative/dbless in this PR and why they are needed for incremental sync? This will help with reviewing.